packages/vaadin-element-mixin/vaadin-element-mixin.d.ts

```diff
@@ -1,5 +1,5 @@
-/// <reference path="custom_typings/vaadin-usage-statistics.d.ts" />
-/// <reference path="custom_typings/vaadin.d.ts" />
+import 'custom_typings/vaadin-usage-statistics.js';
```

Comment:
I'm not quite sure about this change, but TS considers `reference` tags as deprecated and offers to use `imports` instead.